apphost/build.rs

```diff
@@ -9,5 +10,15 @@ fn main() {
         embed_manifest(new_manifest("Piton")).expect("unable to embed manifest file");
     }
 
+
+    if std::env::var_os("CARGO_FEATURE_TESTAPP").is_some() {
+        println!("cargo:rerun-if-changed=test");
+        Command::new("dotnet")

```

Comment:
Not sure if this places the output DLL in the right place (I would expect it to be somewhere in a `bin/` directory); additionally, those output `bin` / `obj` dirs should probably be .gitignored.